src/Microsoft.DotNet.Build.Tasks.SharedFramework.Sdk/targets/framework.sharedfx.targets

```
@@ -251,7 +251,7 @@
     generate shared frameworks in the Arcade SDK.
   -->
   <Target Name="BuildPkgProjectReferences">
-    <MSBuild Projects="$(RepoRoot)signing\SignBinaries.proj" Targets="Build" />
+    <MSBuild Projects="$(SigningToolsDir)SignBinaries.proj" Targets="Build" />
```

When you don't control all the properties yourself the MSBuild helper functions are useful:
```suggestion
    <MSBuild Projects="$([MSBuild]::NormalizePath('$(SigningToolsDir)', 'SignBinaries.proj'))" Targets="Build" />
```

---

This makes sure that directory seperators are placed correctly if missing.

---

Interesting, haven't seen this before. Unfortunate how wordy it is, but nice.